### PR TITLE
feat(swift): generate tests for helpers

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

[DI-2515](https://algolia.atlassian.net/browse/DI-2515)

Generate CTS for swift helpers.

[DI-2515]: https://algolia.atlassian.net/browse/DI-2515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: scripts/cts/runCts.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ async function runCtsOne(language: string): Promise<void> {
       await run('sbt test', { cwd, language });
       break;
     case 'swift':
-      await run('rm -rf .build && swift test -Xswiftc -suppress-warnings -q --parallel', {
```
Comment (shortcuts) [inline scripts/cts/runCts.ts:63]: do we want to cache it in the artifacts/actions? not sure we do

**Review by shortcuts — COMMENTED** [review]
lots of wonderful fixes here, waiting for the codegen

**Review by shortcuts — APPROVED** [review]
super huge, nothing to add on my side

File: scripts/cts/runCts.ts (hunk 1 above)
Comment (millotp) [reply scripts/cts/runCts.ts:63]: yeah it might be helpful, I'll try in another PR

Comment (millotp) [reply scripts/cts/runCts.ts:63]: btw swift is not the slowest, js is the bottleneck

Comment (shortcuts) [reply scripts/cts/runCts.ts:63]: yup the build is slow but it's expected since all clients are doubled

**Pull request merged**